projects/map-ol/src/lib/map-ol.component.ts

```diff
@@ -34,6 +34,8 @@ import olRotate from 'ol/control/Rotate';
 import olMapBrowserEvent from 'ol/MapBrowserEvent';
 import olMapEvent from 'ol/MapEvent';
 import { Control as olControl } from 'ol/control';
+import { GeoJSON } from 'ol/format';
+import VectorSource from 'ol/source/Vector';
```

Comment:
Can you prefix this pleas with 'ol' like the other imports.

Reply:
Updated!